### ISSUE TITLE
Request methods spoofing

### ISSUE THREAD

**Issue by herlon214** [kristijanhusak/laravel-form-builder#366]
Hi guys, there's any implementation about spoofing (as follows https://laravel.com/docs/5.0/routing#method-spoofing) ?

**Comment by Luukvdo** [comment]
Yes, you can pass a method to the form as described [here](http://kristijanhusak.github.io/laravel-form-builder/form/basics.html#form-options).
When setting the method to PATCH/PUT/DELETE the `_method` field will be added to the form automagically.
